docs/semgrep-appsec-platform/webhooks.md

```diff
@@ -14,6 +14,10 @@ import Notifications from "/src/components/concept/_notification-deduplication.m
 
 Webhooks are a generic method for Semgrep AppSec Platform to post JSON-formatted findings after each scan to your URL endpoint.
 
+:::tip For Slack integrations
+- To integrate with Slack, use the [Semgrep Slack integration](/semgrep-appsec-platform/slack-notifications). The webhook set up described in this guide does not apply to Slack integrations.
```

Comment:
```suggestion
- To integrate with Slack, use the [Semgrep Slack integration](/semgrep-appsec-platform/slack-notifications). The webhook setup described in this guide does not apply to Slack integrations.
```

I thought we preferred "Semgrep Slack app" for the app?

Reply:
![Snag_1160111](https://github.com/user-attachments/assets/f54d3cce-ae10-4a49-a8d6-62b06642a087)
Ya, thanks for the reminder!